docs/_config.yml

```diff
@@ -60,8 +60,8 @@ asset_urls:
 aux_links:
   "Ask a Question":
     - "https://discourse.firez.one/new-topic?body=%23%23%20Describe%20the%20issue%0A%0AA%20clear%20and%20concise%20description%20of%20what%20the%20bug%20is.%20Code%20samples%20are%20helpful.%0A%0A%23%23%20Reproduction%20instructions%0A%0ATry%20to%20give%20as%20much%20context%20as%20you%20can!%20%0A%0A%23%23%20Expected%20behavior%0A%0ADescription%20of%20expected%20behavior.%0A%0A%23%23%20Screenshots%0A%0AIf%20applicable%2C%20add%20screenshots%20to%20help%20explain%20your%20problem.%0A%0A%23%23%20Environment%20Details%0A%0A-%20OS%3A%20%5Be.g.%20Ubuntu%2C%20Fedora%2C%20CentOS%5D%0A-%20VPC%3A%0A-%20Firebase%20Version%3A%20%0A%0A%23%23%20Other%20Info%0A%0AAdd%20any%20other%20context%20about%20the%20problem%20here.&category=category/subcategory&tags=docs"
-  "Schedule a Demo":
-    - "https://e04kusl9oz5.typeform.com/to/BkWP2ys1"
+  "Join the Beta":
+    - "https://e04kusl9oz5.typeform.com/to/uD689MvN#source=docs"
```

Comment:
Just a sanity check here -- should this be using the standard URL param delimiter -- `?` instead of `#`?

Reply:
both seem to work. For some reason typeform's default is `#`